src/components/PreFooter.tsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+import Image from 'next/image';
```

Comment:
We need to use our custom `Image` component in order to remove the `style="color:transparent"` the Next.js component adds to image.